src/index.html

```diff
@@ -10,8 +10,145 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
```

Comment:
Follow the requirements, please 
On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application